### PR TITLE
v1.2.0: Danger.js has been added to do basic PR checks

### PR DESCRIPTION
**Pull request by ashleynolan**
### Added
-   Danger.js has been added to do basic PR checks.  This will leave comments on the PR when the `CHANGELOG` hasn’t been updated, the version needs incrementing or if it thinks the `README` might need an update.
-   Adding Github PR template and CODEOWNERS files
-   Fozzie icon to README (very important!)

### Changed
-   Dependencies updated to latest versions

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (DamianMullins) [inline .babelrc]: Is this file required?

Comment (ashleynolan) [reply .babelrc]: Unfortunately, it's a `babel-jest` side issue that [needs to be worked around](https://github.com/danger/danger-js/issues/261).
